src/main/java/no/nav/pto/veilarbportefolje/client/RestClientUtils.java

```diff
@@ -1,17 +1,18 @@
 package no.nav.pto.veilarbportefolje.client;
 
-import no.nav.common.auth.context.AuthContextHolder;
+import no.nav.common.auth.context.AuthContextHolderThreadLocal;
 
 public class RestClientUtils {
 
-        public static String authHeaderMedSystemBruker() {
-            return AuthContextHolder.getIdTokenString()
-                    .map(RestClientUtils::createBearerToken)
-                    .orElseThrow(IllegalStateException::new);
-        }
+    public static String authHeaderMedSystemBruker() {
+        return AuthContextHolderThreadLocal
+                .instance().getIdTokenString()
+                .map(RestClientUtils::createBearerToken)
+                .orElseThrow(IllegalStateException::new);
+    }
```

Comment:
Comment from Alexander: Nja, det vil fungere, men tanken bak AuthContextHolderThreadLocal er at den skal lages som en bønne og injectes inn de stedene det trengs. Grunnen til en bønne fungerer bedre er fordi da har du mulighet til å mocke ut AuthContextHolderThreadLocal når man lager tester

Reply:
@Bean
    public AuthContextHolder authContextHolder() {
        return AuthContextHolderThreadLocal.instance();
    }

Reply:
Yep, I will try to improve that...although its not everywhere possible to inject bean

Reply:
true and he said that i should work so there is no worry :)